pro-planner-client/src/routes/VotePage.jsx

```diff
@@ -21,8 +22,10 @@ const VotePage = () => {
 
     return <>
         {(loadingState === LOAD_STATUS.LOADING) && <LoadingDisplay/>}
```

Comment:
Should the polls component be rendered if loading status is still set to loading? From this, it seems like both components will be shown while loading occurs.